### PR TITLE
hatchback

### PR DESCRIPTION
**Pull request by AlisGraveNil**
## Description

This is a TKL DIY solder kit that can be converted into a 75% keyboard + a 13-key macropad.
This firmware is only for the TKL/75% and only one .hex needs to be created to control both; the firmware for the macropad will be in a different PR.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [X] Keyboard (addition or update)
- [X] Keymap/layout/userspace (addition or update)
- [X] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [X] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [X] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [X] My change requires a change to the documentation.
- [X] I have updated the documentation accordingly.
- [X] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [X] I have added tests to cover my changes.
- [X] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Review by drashna — CHANGES_REQUESTED** [review]
Could you rename the folder to only be lower case? 

Also, you may need to rebase to fix the git submodules (the `lib` folder) changes.

**Comment by AlisGraveNil** [comment]
@drashna - I think I got the rebase right; I can't tell if trying to lower-case it took

**Comment by drashna** [comment]
submodules fixed, but folder name needs the temp name treatment, it looks like.

**Comment by AlisGraveNil** [comment]
well... I corrected the name, but for some reason it wants to rewrite a bunch of other files too...

**Comment by AlisGraveNil** [comment]
or maybe now it's fixed?

**Review by drashna — COMMENTED** [review]
Just missing a readme

File: keyboards/clawsome/hatchback/readme.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# SUV
+
+This is TKL keyboard that can be converted down into a 75% keyboard and a 13key macropad (see "luggage rack")
+
+- Keyboard Maintainer: [AAClawson](https://github.com/AlisGraveNil)
+- Hardware Supported: Hatchback, Elite-C (as TKL); Pro-Micro and nice!nano (as 75%)
+- Hardware Availability: In stock within the next month
+
+Make example for this keyboard (after setting up your build environment):
```
Comment (drashna) [inline keyboards/clawsome/hatchback/readme.md:9]: Could you add instructions on how to enter the bootloader to flash new firmware?

**Review by drashna — APPROVED** [review]

**Review by fauxpark — APPROVED** [review]

**Comment by AlisGraveNil** [comment]
@drashna - can you push this to merge to the Master, since it's gotten two approval checks?

**Comment by drashna** [comment]
Thanks!

**Pull request merged**